internal/dao/pod.go

```diff
@@ -330,6 +330,38 @@ func (p *Pod) Scan(ctx context.Context, gvr client.GVR, fqn string, wait bool) (
 	return refs, nil
 }
 
+// GetOwners returns the owners of the resource.
+func (p *Pod) GetOwners(path string) ([]OwnerInfo, error) {
+	var owners []OwnerInfo
+
+	pod, err := p.GetInstance(path)
+	if err != nil {
+		return nil, err
+	}
+	ownerRefs := pod.GetObjectMeta().GetOwnerReferences()
```

Comment:
I think we should DRY this up and extract this into a convenience call to gather controllers gvr/fqn from OwnerRefs